docs/basic-docs/atomicdex/atomicdex-api-20/get_public_key.md

```diff
@@ -0,0 +1,53 @@
+## get\_public\_key
+
+**get_public_key**
+
+The `get_public_key` method returns the pubkey corresponding to the user's seed phrase.
```

Comment:
Could you please add `the compressed secp256k1 pubkey` - it will be a bit more clear :slightly_smiling_face: